README.md

```diff
@@ -1,3 +1,68 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/distroless.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/distroless)
 
-# This repository contains source code used to build "distroless" docker images.
+# "Distroless" Docker Images
+
+"Distroless" images contain only your application and it's runtime dependencies.
+They do not contain package managers, shells any other programs you would expect to find in a standard Linux distribution.
+
+For more information, see this talk.
+
+## Why should I use distroless images?
```

Comment:
Restricting what's in your runtime to container to precisely what's necessary for your app is a best practice employed by Google and other tech giants that have used containers in production for many years.  It improves the signal to noise of scanners (e.g. CVE) and reduces the burden of establishing provenance to just what you need.